src/server/input/default_input_device_hub.cpp

```diff
@@ -39,6 +40,26 @@
 
 namespace mi = mir::input;
 
+namespace
+{
+class DefaultInputDeviceHubKeyMapperObserver : public mi::KeyMapperObserver
```

Comment:
I don't think the DefaultInputDeviceHub should be registering. Only devices that support LEDs